example/components/weather-forecast.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useSynapse } from '../synaptik';
+
+export default () => {
+  const [forecast, loading, load] = useSynapse(
```

Comment:
Are we expecting to have type check here?  For example, `const [forecast, loading, load, foo] = useSynapse(` or `console.log(forecast.abc` would throw an error.

Reply:
@vietnogigoodrx are you asking if we have a typecheck for providing a selector function as the first argument to `useSynapse`?

Reply:
It appears that there are no typing checking here and I was wondering if we should have it.

Reply:
@vietnogigoodrx if you pull down and view the example in your editor, you'll see that typings are present. The point of this PR is to make typings invisible and inferred based on the store instances 😄

Reply:
<img width="986" alt="Screen Shot 2020-05-18 at 11 41 08 AM" src="https://user-images.githubusercontent.com/6404663/82248149-7f1cdd00-98fc-11ea-852f-6e3c410e95f8.png">
<img width="633" alt="Screen Shot 2020-05-18 at 11 41 32 AM" src="https://user-images.githubusercontent.com/6404663/82248173-8b089f00-98fc-11ea-9d1c-428705b7fb30.png">

Reply:
Restarted my vscode and typings are there now.  Very nice! 🎉